packages/docs/src/examples/ConnectingToNetwork.tsx

```diff
@@ -21,17 +22,54 @@ ReactDOM.render(
 const STAKING_CONTRACT = '0x00000000219ab540356cBB839Cbe05303d7705Fa'
 
 export function App() {
-  const { activateBrowserWallet, deactivate, account } = useEthers()
+  const { account, activateBrowserWallet, deactivate } = useEthers()
   const userBalance = useEtherBalance(account)
   const stakingBalance = useEtherBalance(STAKING_CONTRACT)
+
+  const ConnectButton = () => (
+    <div>
+        <button onClick={() => activateBrowserWallet()}>Connect</button>
+        <p>Connect to wallet to interact with the example.</p>
+    </div>
+  )
+
+  const MetamaskConnect = () => (
+    <div>
+      {account && (
+      <div>
+          <div id="Inline_TYMo">
```

Comment:
This `_xxxx` looks bad. It looks ugly, and it can change in the future.
We should change webpack to not add it anymore.